### DISCUSSION TITLE
Is there any chance that sub-pixel hinting will be implemented?

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/ecode#159] [General]
If you ask me what kind of hinting I want like on https://github.com/SpartanJ/ecode/discussions/94, I can't answer. If you said you were not very versed in the matter, then I'm completely ignorant. The only thing I know is that, out of the box, the font rendering on Pragtical is better on my screen. Increasing the font size on ecode makes things a lot better.

**Comment by SpartanJ** [comment]
I personally don't like sub-pixel hinting, usually font clarity can be improved with a different / nicer font. Implementing sub-pixel hinting is complicated and I lack the interest to do it. I understand this is something very personal but the whole industry is moving away from sub-pixel hinting given that screens are improving its average pixel density over time.

**Reply by ghost** [reply]
How can Lite XL support sub-pixel hinting? They are using SDL2 like you. Perhaps you could learn something from them. Btw, I don't need sub-pixel hinting. I'm OK with the current font rendering.

**Reply by SpartanJ** [reply]
It has nothing to do with SDL2. Lite-XL doesn't use OpenGL or any GPU acceleration for rendering, it just renders to a software frame buffer, which makes sub-pixel hinting really easy.
